install/operator/pkg/syndesis/olm/docker_test.go

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2020 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package olm
+
+import (
+	"context"
+	"testing"
+
+	"github.com/syndesisio/syndesis/install/operator/pkg/apis/syndesis/v1beta1"
+	"github.com/syndesisio/syndesis/install/operator/pkg/syndesis/configuration"
+	syntesting "github.com/syndesisio/syndesis/install/operator/pkg/syndesis/testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_docker_build(t *testing.T) {
+	clientTools := syntesting.FakeClientTools()
+	conf, err := configuration.GetProperties(context.TODO(), "../../../build/conf/config-test.yaml", clientTools, &v1beta1.Syndesis{})
```

Comment:
You've included a clientTools in your _conf_ here but not used in that actual implementation. Probably makes little difference if d.build() doesn't use it but might be problematic in the future?
Might be safer for them to be consistent since the test would at least flag up the same result as the implementation.

Reply:
You mean I dont need the clienttool and it is enough with nil?